sievelib/commands.py

```diff
@@ -405,7 +418,7 @@ def check_next_arg(self, atype, avalue, add=True, check_extension=True):
                         if not curarg["name"] in self.arguments:
                             self.arguments[curarg["name"]] = []
                         self.arguments[curarg["name"]] += [avalue]
-                elif atype not in curarg["type"] or \
+                elif self.__is_valid_type(atype, curarg["type"]) or \
```

Comment:
```suggestion
                elif not self.__is_valid_type(atype, curarg["type"]) or \
```
According to my previous comment.